src/pages/tokens/[id].tsx

```diff
@@ -0,0 +1,203 @@
+import { GetServerSidePropsContext, GetServerSidePropsResult, InferGetServerSidePropsType } from 'next'
+import { getWhaleApiClient } from '@contexts/WhaleContext'
+import { TokenData } from '@defichain/whale-api-client/dist/api/tokens'
+import { AdaptiveList } from '@components/commons/AdaptiveList'
+import { Breadcrumb } from '@components/commons/Breadcrumb'
+
+import { getAssetIcon, getTokenIcon } from '@components/icons/assets'
+import { IoAlertCircleOutline, IoCheckmarkCircleOutline, IoCopyOutline } from 'react-icons/io5'
+import { PoolPairsTable } from '@components/commons/tables/PoolPairs'
+
+interface TokenAssetPageProps {
+  token: TokenData
+}
+
+export default function TokenAssetPage (props: InferGetServerSidePropsType<typeof getServerSideProps>): JSX.Element {
```

Comment:
Remove this?

Reply:
This is the exported component. It can not be removed. :)